### PR TITLE
next: rollout 32.20200923.1.0

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

File: updates/next.json [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
       }
     },
     {
-      "version": "32.20200907.1.1",
+      "version": "32.20200923.1.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 1440,
-          "start_epoch": 1600439400,
+          "start_epoch": 1600974000,
```
Comment (dustymabe) [inline updates/next.json:28]:
```
$ date -d @1600974000
Thu Sep 24 03:00:00 PM EDT 2020
```

+1

File: updates/next.json [hunk 2]
```diff
@@ -13,19 +13,19 @@
...
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "32.20200907.1.1",
+      "version": "32.20200923.1.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 1440,
```
Comment (dustymabe) [inline updates/next.json:27]: shall we bump this back up to 2880? We bumped it down for the release earlier this week but we should probably go back to 48H as a standard.

Comment (cverna) [reply updates/next.json:27]: Sure will do

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by jlebon — APPROVED** [review]
```
next
    version: 32.20200923.1.0
    start: 2020-09-24 19:00:00+00:00 UTC (in 0:55:15)
    duration: 2880m (48.0h)
```

**Pull request merged**